### PR TITLE
Fix(grid): epsg projection information and length_multiplier (issue #532)

### PR DESCRIPTION
**Pull request by jlarsen-usgs**
* Fix for epsg projection information to write to modflow name file

* Added length_multiplier parameter into Grid classes

* updated t007_test.test_epsgs() to test for epsg projection information 

* updated t007_test.test_mg() to test the length_multiplier parameter

Issue #532

### CODE REVIEW

**Comment by langevin-usgs** [comment]
@jlarsen-usgs, thanks for looking into this.  And I'm sorry to ask this question now, but is length_multiplier needed?  And if so, what is it's purpose?  I know that @jdhughes-usgs mentioned this in the issue, but I'm guessing his question was more related to it's purpose in the name file.  The whole length_multiplier is one of the tricky and confusing parts of spatial reference.   My understanding of it's intent is that one could build a model in a projection that has feet, but have the model input be in meters.  Is this something that we want to support?  If we do support it, then that's where the confusion enters.  What are the plots in?  Are the values of delr.array in model units or projection units?  I think we are asking for trouble if we support it.   That's my 2 cents but maybe I'm missing something.

**Comment by jdhughes-usgs** [comment]
Lets make a decision on `length_multiplier` before accepting the pull request. I would be in favor of eliminating `length_multiplier` unless there is a need for it. I suspect the only need for it would be for netcdf export, which exports in lat/lon, when the length units are undefined...but the projection information should take care of this issue as long as the projection is consistent with the length units in the model.

**Comment by langevin-usgs** [comment]
I think when we went into this revamping/deprecation of spatial reference, we were thinking that the user would have to have the same length units for the model that they have for the projection.  With this assumption, we remove most of the complications that were were having with SR.  In the future, we could support exporting or perhaps even plotting in a different projection, but that would be different functionality that would not require storing any sort of length_multiplier with the grid itself.

**Comment by spaulins-usgs** [comment]
The Grid classes had length_multiplier in an early "alpha" version. After some discussion we decided to remove length_multiplier since it added unnecessary complexity. I am also in favor of keeping it out of Grid unless there is a need for it.

**Comment by jlarsen-usgs** [comment]
Should we remove the `length_multiplier` entry from the nam file header as well then? I don't see a reason to continue writing into the header if we have decided to no longer support it.

**Comment by jdhughes-usgs** [comment]
I think so.

File: flopy/discretization/grid.py [hunk 1]
```diff
@@ -50,6 +50,8 @@ class Grid(object):
         in the spatial reference coordinate system
     rotation : float
         rotation angle of model grid, as it is rotated around the origin point
+    length_multiplier : float
```
Comment (langevin-usgs) [inline flopy/discretization/grid.py]: Should length_multiplier be removed from docstrings?

Comment (jlarsen-usgs) [reply flopy/discretization/grid.py]: Yes it should! Thanks for catching that.

**Pull request merged**